themes/inpycon2017/static/css/styles.css

```diff
@@ -335,6 +335,99 @@ pre code
 	text-align: center;
 	background: #484848;
 }
+ul {
+    margin: 0px;
+    padding: 0px;
+}
```

Comment:
Let's not apply any global rule here. If there is a specific section of page where this is applicate, it would be scoped with a class name and the rule should be applied.